pyresttest/validators.py

```diff
@@ -1,5 +1,6 @@
 import logging
 import json
+import Cookie
```

Comment:
Even easier, just do this:
```from .six.moves import http_cookies as Cookie
```
I'm testing this patch with the above and it works perfectly well on both Py2 and Py3:
```
$ python2 $(which pyresttest) http://localhost:8000 test/api.yml 
Test Group Default SUCCEEDED: : 5/5 Tests Passed!

$ python3 $(which pyresttest) http://localhost:8000 test/api.yml 
Test Group Default SUCCEEDED: : 5/5 Tests Passed!
```